pkg/operation/botanist/component/seedadmissioncontroller/seedadmissioncontroller.go

```diff
@@ -456,7 +456,7 @@ func GetValidatingWebhookConfig(caBundle []byte, webhookClientService *corev1.Se
 			SideEffects:             &sideEffect,
 			TimeoutSeconds:          pointer.Int32(10),
 		}, {
-			Name: "validation.extensions.seed.admission.core.gardener.cloud",
+			Name: "validation.extensions.etcd.admission.core.gardener.cloud",
```

Comment:
Why is this one special and not contained in `getWebhooks`?

Reply:
This is fine here, As it has many different things than others.

Reply:
If we keep this also in `getwebhook` then we need to use `if else` in `APIGroup` and `APIVersion`, that kind of a little less readable.